qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature

```diff
@@ -14,4 +14,4 @@ Feature: Reports Management
     And user enters start date
     And user enters end date
     And user clicks on request report button
-    #Then user clicks on view report link
+    #Then user clicks on view report link
```

Comment:
is this change part of the ticket requirement?

Reply:
checkout this line on master branch.   these were brought about updating my branch to master  https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature#L17